menus/setup-verifier.js

```diff
@@ -120,39 +123,37 @@ async function getDomainLinkCredential(keypairs, origin, account) {
   })
 
   const claimContents = {
-    id: fullDid.did,
+    id: fullDid.uri,
     origin: origin,
   }
 
   const claim = Claim.fromCTypeAndClaimContents(
     ctype,
     claimContents,
-    fullDid.did
+    fullDid.uri
   )
 
   const request = RequestForAttestation.fromClaim(claim)
 
-  const { signature, keyId } = await fullDid.signPayload(
-    Utils.Crypto.coToUInt8(request.rootHash),
+  const selfSignedRequest = await request.signWithDidKey(
     {
       async sign({ data, alg }) {
-        const { assertion } = keypairs
+        const { authentication } = keypairs
         return {
-          data: assertion.sign(data, { withType: false }),
```

Comment:
what is the story behind having to change the key pair we're using?

Reply:
It wouldn't work if I ran the assertion vs authentication.